custom-words.txt

```diff
@@ -1385,6 +1385,7 @@ offboard
 offboards
 officedocument
 OLTP
+OIDC
```

Comment:
Just curious, why we need put OIDC in this file?

Reply:
it was failing because of the spelling check